Gemfile

```diff
@@ -42,7 +42,6 @@ gem "acts_as_tree",                   "~>2.1.0"  # acts_as_tree needs to be requ
 # https://github.com/jeremyevans/ruby-american_date
 gem "american_date"
 gem "azure-armrest",                  :git => "git://github.com/ManageIQ/azure-armrest.git"
-gem "bcrypt",                         "3.1.10"
```

Comment:
please leave in Gemfile

If it is required by our app, it should be in gemfile.